Sources/Articles/Documentation.docc/2025/jwt-kit/jwt-kit.md

```diff
@@ -0,0 +1,198 @@
+# Getting Started with Vapor's JWTKit 5: A Practical Introduction to JSON Web Tokens in Swift
+
+JWTKit is a library for working with JSON Web Tokens (JWT) in Swift. It provides a simple and easy-to-use API for creating, parsing, and validating JWTs. JWTs are a popular way to securely transmit information between parties, and JWTKit makes it easy to work with them in your Swift projects.
```

Comment:
```suggestion
JWTKit is a library for working with JSON Web Tokens (JWT) in Swift. It provides a simple and easy-to-use API for creating, parsing, and validating JWTs. JWTs are a popular way to authenticate information (claims) between parties. JWTKit makes it easy to work with them in your Swift projects.
```

Reply:
It's not secure in the sense people would expect. JWEs do encrypt data, but a common JWT (JWS) only signs data. It's not "secure" in the sense of unreadable.